README.md

```diff
@@ -1974,6 +1978,15 @@ and 'default'.
 Enable/Disable negotiate auto on the interface. Valid values are 'true',
 'false', and 'default'.
 
+##### `storm_control_broadcast`
+Allowed broadcast traffic level. Valid values are string, keyword 'default'.

```

Comment:
I think it would be helpful to indicate what the string represents.  `Valid values are a string representing the broadcast level or keyword 'default'.`